.readthedocs.yaml

```
@@ -0,0 +1,10 @@
+version: 2
+
+build:
+  os: "ubuntu-20.04"
```

Any reason not use `ubuntu-22.04`
```suggestion
  os: "ubuntu-22.04"
```